packages/mobile/src/geth/saga.ts

```diff
@@ -129,6 +133,12 @@ function createNewBlockChannel() {
 function* monitorGeth() {
   const newBlockChannel = yield createNewBlockChannel()
 
+  if (isZeroSyncMode()) {
+    yield put(setGethConnected(true))
```

Comment:
Should we have a TODO here to check the health of the connection to the tx node or something like that?

Reply:
In `src/transactions/send.ts`, I added a dev-mode only check which verifies that the connection is possible.  There isn't much advantage of doing this the first time, since it can still fail later and we are not really syncing anyways.

Reply:
Ok